kubernetes/windows/setup.ps1

```diff
@@ -64,6 +67,22 @@ Write-Host ('Extracting Certificate Generator Package')
     Expand-Archive -Path /opt/amalogswindows/certificategenerator.zip -Destination /opt/amalogswindows/certgenerator/ -Force
 Write-Host ('Finished Extracting Certificate Generator Package')
 
+Write-Host ('Installing Windows Azure Monitor Agent');
+try {
+    $windowsazuremonitoragent='https://github.com/microsoft/Docker-Provider/releases/download/windows-ama-bits/genevamonitoringagent.46.2.1.zip'
```

Comment:
Is this agent build official build from AMA team which we can go?

Reply:
This is the official build but all changes needed are not present in this one for Perf table.